### PR TITLE
homework_16 with completed task

### PR DESCRIPTION
**Pull request by akachanova**

### CODE REVIEW

File: homework-16/calculator.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    return firstNumber - secondNumber;
+}
+
+export function calculatorDivision(firstNumber: number, secondNumber: number) {
+    return firstNumber / secondNumber;
+}
+
+export function calculatorMultication(firstNumber: number, secondNumber: number) {
+    return firstNumber * secondNumber;
+}
```
Comment (VladSelikh) [inline homework-16/calculator.ts]: Please rewrite these function into a `class` with the corresponding methods.

File: homework-16/custom.checker.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  calculatorMinus,
+  calculatorDivision,
+  calculatorMultication,
+} from "./calculator";
+
+export function customChecker(
+  firstNumber: number,
+  secondNumber: number,
+  result: number,
+  sign: string
```
Comment (VladSelikh) [inline homework-16/custom.checker.ts]: Please be more specific in the `sign` variable possible values using TypeScript syntax.

File: homework-16/index.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { customChecker} from "./custom.checker";
+
+//positive tests7
+customChecker(1, 2, 3, '+');
+customChecker(7, 2, 14, '*');
+customChecker(9, 3, 3, '/');
+customChecker(100, 70, 30, '-');
+customChecker(44, 44, 88, '+');
```
Comment (VladSelikh) [inline homework-16/index.ts]: Please use test design technics while writing your tests. For example it would be nice to add tests for adding `0` to a positive number, adding two negative numbers while testing addition method.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "tms",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "intro": "ts-node ./homework-16/index.ts"
```
Comment (VladSelikh) [inline package.json]: Please rename your script as `hw16`.

File: homework-16/calculator.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+    constructor(firstNumber: number, secondNumber: number) {
+        this.firstNumber =  firstNumber;
+        this.secondNumber = secondNumber;
+    }
+
+    multiplication() {
+        return this.firstNumber * this.secondNumber;
+    }
+}
```
Comment (VladSelikh) [inline homework-16/calculator.ts]: Please create a function `calculator` that receives 2 numbers, and the third parameter is an operation.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "tms",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "intro": "ts-node ./homework-16/hw16.ts"
```
Comment (VladSelikh) [inline package.json]: Please rename the script as `hw16` for example.
